contributors/guide/issue-triage.md

```diff
@@ -4,253 +4,291 @@ weight: 10
 
-## Poke SIG if a SIG label is assigned but no comment was added by SIG in 30 days
+### If a SIG label Is Assigned, but No Action Is Taken Within 30 Days
+If you find an issue with a SIG label assigned, but there's no evidence of movement or discussion within 30 days, then gently poke the SIG about this pending issue. Also, consider attending one of their meetings to bring up the issue, if you feel this is appropriate.
 
-Ideally the SIG lead should have a SIG member that is a first point
-of contact for SIG new issues. If an issue has a SIG label assigned and no
-action is taken by SIG in 30 days (e.g. no comment was added by SIG or no
-discussion was initiated) then gently poke SIG about this pending issue.
-Also, consider attending one of the SIG meetings and bringing up the issue, if
-you feel this is appropriate.
+### If an Issue Has No Activity After 90 Days
+When this happens, the `fejta-bot` adds the `lifecycle/stale` label to that issue. You can block the bot by applying the `/lifecycle frozen` label preemptively, or remove the bale with the `/remove-lifecycle stale` label. The `fejta-bot` adds comments in the issue that include additional details. If you take neither step, the issue will eventually be auto-closed. 
 
-## Milestones
+To gain a better understanding of an issue triaged by the `fejta-bot`, so that you can close it, you can add any of the `triage/*` labels described above.
 
-We additionally use milestones, based on minor version, for determining if a bug
-should be fixed for the next release. These milestones will be especially
-scrutinized as we get to the weeks just before a release. We can release a new
-version of Kubernetes once they are empty. We will have two milestones per minor
-release.
+## Planning Milestones
+Please visit the developer guide section for detailed information on [targeting features, issues and PRs to release milestones](https://github.com/LappleApple/community/blob/master/contributors/devel/sig-release/release.md). We use milestones, based on minor version, for determining if a bug should be fixed as part of the next Kubernetes release. Every minor release has two milestones, and they are increasingly scrutinized in the weeks just before a release. When our milestones are empty, we can release a new version of Kubernetes.
 
+### Some Pointers About Bugs
 - **vX.Y**: The list of bugs that will be merged for that milestone once ready.
+- **vX.Y-candidate**: The list of bugs that we might merge for that milestone. A bug shouldn't be in this milestone for more than a day or two towards the end of the milestone's lifespan. It should either be triaged either into vX.Y, or moved out of the release milestones.
+- The above [priority](#define-priority) scheme still applies. 
+- A few days before release, we will probably move all `priority/important-soon` and `priority/important-longterm` bugs out of that milestone in bulk.
```

Comment:
I would cut this section following above.

Reply:
Thanks for this and the above suggestions -- plan to work on it later today. :)